core/store/models/job_spec.go

```diff
@@ -313,3 +314,28 @@ func (t *TaskType) Scan(value interface{}) error {
 	*t = TaskType(temp)
 	return nil
 }
+
+// LinkEarned is to track Chainlink earnings of individual
+// job specs from job runs
+type LinkEarned struct {
+	ID        uint64       `json:"id,omitempty" gorm:"primary_key;not null;auto_increment"`
```

Comment:
Let's remove `omitempty`. This should never be empty when sending outside of core, no?